index.js

```diff
@@ -118,23 +119,20 @@ for (let index = 0, retry = 0; index < mysql.tenantUUIDs.length; index++) {
 
       retry = 0;
    } catch (error) {
-      healtcheck = `${process.env.HEALTH_CHECK_URL}/fail`;
+      errorMessages.push(error.message || String(error);
```

Comment:
Do we need another close bracket )? 

Is it better if we use ?? instead of ||

Reply:
Yes, thank you.